scipy/stats/_continuous_distns.py

```diff
@@ -517,15 +517,18 @@ def _entropy(self):
 
 class FitDataError(ValueError):
     # This exception is raised by, for example, beta_gen.fit when both floc
-    # and fscale  are fixed and there are values in the data not in the open
-    # interval (floc, floc+fscale).
-    def __init__(self, distr, lower, upper):
-        self.args = (
-            "Invalid values in `data`.  Maximum likelihood "
-            "estimation with {distr!r} requires that {lower!r} < x "
-            "< {upper!r} for each x in `data`.".format(
-                distr=distr, lower=lower, upper=upper),
-        )
+    # and fscale are fixed and there are values in the data not in the open
+    # interval (floc, floc+fscale), or with a custom message.
+    def __init__(self, distr=None, lower=None, upper=None, msg=None):
+        if msg:
+            self.args = (msg,)
+        else:
+            self.args = (
+                "Invalid values in `data`.  Maximum likelihood "
+                "estimation with {distr!r} requires that {lower!r} < x "
```

Comment:
Sam, go ahead and change this.
```suggestion
                "estimation with {distr!r} requires that {lower!r} < (x - loc)/scale "
```